### DISCUSSION TITLE
Explainable AI

### DISCUSSION THREAD

**Discussion by Meddebma** [Project-MONAI/MONAI#2450] [General]
A big problem of AI in medicine is that "black box" approach is unacceptable for clinical decision support. I would like to work with you on a solution to integrate not only radiological Data but also clinical data to predict treatment's outcomei. 
One possible solution would be to change classification model architechture (like UNET) to insert 2D or 3D images, but also patient's clinical data (Like Age, Gender, Smoker, inflammation lab results etc..) to predict not only a class, but also to find out which clinical data were the most important to classification (something like feature importance and ranking) and thus make AI more explainable. Here is an example of research:
https://proceedings.neurips.cc/paper/2017/file/8a20a8621978632d76c43dfd28b67767-Paper.pdf
Let me know if you are interested in this.
Thanks

**Comment by Nic-Ma** [comment]
Hi @Meddebma ,

Thanks for your interest here.
@ericspod @wyli May I know is there any multimodal discussion in our working group related to this ticket? I would mark this ticket as `working group discussion`.

Thanks.

**Reply by rijobro** [reply]
hi @Meddebma, @Nic-Ma, I've moved this from a tutorials issue to a discussion on the main repo.

Related discussion – https://github.com/Project-MONAI/MONAI/discussions/2212.
